src/controllers/portfolio/portfolio.ts

```diff
@@ -404,7 +418,80 @@ export class PortfolioController extends EventEmitter {
     this.emitUpdate()
   }
 
-  // NOTE: we always pass in all `accounts` to ensure that the user of this
+  protected async updatePortfolioState(
+    accountId: string,
+    _accountState: AccountState,
+    network: Network,
```

Comment:
Isn't the naming convention `#updatePortfolioState` for private methods?

Reply:
it's a compromise, because a private method can't be mocked with Jest, will add a comment